spec/dish_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'dish'
+
+describe Dish do
+  describe '#initialised' do
+
+    before(:each) do
```

Comment:
Try to use let instead of before, as its a bit more efficient in terms of resources required.

Why have you used an instance variable (@dish) here?